devel/glib2/Portfile

```diff
@@ -15,7 +15,7 @@ name                        glib2
 conflicts                   glib2-devel
 set my_name                 glib
 version                     2.78.4
-revision                    1
+revision                    2
```

Comment:
Should the py-packaging dependency not have been removed from the port?

Reply:
Yes, it should have. Sorry.

Reply:
> Should the py-packaging dependency not have been removed from the port?

I'll add a reminder comment, and take care of it during the next update.

Reply:
You can remove it now without a rev bump with no ill effects. The dependency will be removed from the registry if upgrade is run on glib2 (which happens when installing or upgrading any dependent), and if that never happens, it just means there's an unused dependency just like if you did nothing.